### DISCUSSION TITLE
Multiple Assignment

### DISCUSSION THREAD

**Discussion by azeem-hussain-ad** [coderflexx/laravel-ticket#64] [Ideas]
What if i have to assign a task to multiple users?
lets say when task is created, i assign that task to a user and he will provide me explanation on it once it is closed then i need to assign that task to a user who will run compaign against that task. so see we are assigning same task to multiple users.

**Comment by ousid** [comment]
Hi, @azeem-hussain-ad 

Have you considered adding [Many to Many relationship](https://laravel.com/docs/11.x/eloquent-relationships#many-to-many-polymorphic-relations)?

Something like `ticket_user` table, then override the `Ticket`, and the `User` models.

This package, is a back-end focused, so you have the ability to change whatever you want, and make it fits on your application.

Let me know, if you have any questions.
